src/components/Dashboard/StaffNotation.vue

```diff
@@ -66,7 +66,7 @@
             <v-list-item
               data-type="registrars-notation"
               @click="showRegistrarsNotationDialog()"
-              :disabled="disabled"
+              :disabled="!isAllowed(AllowableActions.REGISTRARS_NOTATION)"
```

Comment:
This is cleanup and more concise (ie, driven by BE logic).

Basically, I display all items that are applicable for the current state (active or historical), and then disable the ones that aren't allowed.